feeding/src/Workspace.cpp

```diff
@@ -52,7 +52,7 @@ void Workspace::addToWorld(
 {
   Eigen::Isometry3d pose = robotPose.inverse()
                            * createIsometry(getRosParam<std::vector<double>>(
-                               "/" + name + "/pose", mNodeHandle));
+                                 "/" + name + "/pose", mNodeHandle));
```

Comment:
This kind of reformatting happens in a lot of places, I assume this is from a new version of make format or something like that? 😮

Reply:
We recently up-ed to `clang-format-6.0` to match aikido/libada. Sorry, thought that was pushed before, surprised to see this.